.github/workflows/lint-helm.yaml

```diff
@@ -0,0 +1,23 @@
+name: Lint Helm Charts
+on:
+  pull_request:
+    branches:
+      - master
+  pull_request_review:
+    types: [submitted, edited]
+  release:
+    types:
+    - published
+    - created
+    - edited
+jobs:
+  lint-helm:
+    runs-on: ubuntu-latest
```

Comment:
Mind setting this to ubuntu-20.04 for now, and we can update explicitly later when a new version is available?

Reply:
Done :)